core/src/main/java/org/apache/calcite/adapter/enumerable/RexToLixTranslator.java

```diff
@@ -376,6 +394,17 @@ private Expression converterToTime(RelDataType sourceType, Expression operand) {
           BuiltInMethod.TIMESTAMP_WITH_LOCAL_TIME_ZONE_TO_TIME.method,
           operand,
           Expressions.call(BuiltInMethod.TIME_ZONE.method, root)));
+      break;
+    // SQL 2011 Part2 Section 4.6 General Rules 15.e
```

Comment:
I didn't find "general rules" under section 4.6 (was looking at page no.36)

`SQL 2011 Part2 Section 4.6 Table 3` gives a very clear conversion table.

Reply:
Sorry, it should be Section 6.13 (begin with p243)